util/libcrypto.num

```diff
@@ -4514,16 +4514,6 @@ EVP_PKEY_new_CMAC_key                   4455	1_1_1	EXIST::FUNCTION:
 EVP_PKEY_asn1_set_set_priv_key          4456	1_1_1	EXIST::FUNCTION:
 EVP_PKEY_asn1_set_set_pub_key           4457	1_1_1	EXIST::FUNCTION:
 RAND_DRBG_set_defaults                  4458	1_1_1	EXIST::FUNCTION:
-SM2_decrypt                             4459	1_1_1	EXIST::FUNCTION:SM2
-SM2_do_sign                             4460	1_1_1	EXIST::FUNCTION:SM2
-SM2_compute_userid_digest               4461	1_1_1	EXIST::FUNCTION:SM2
-SM2_encrypt                             4462	1_1_1	EXIST::FUNCTION:SM2
-SM2_ciphertext_size                     4463	1_1_1	EXIST::FUNCTION:SM2
-SM2_verify                              4464	1_1_1	EXIST::FUNCTION:SM2
-SM2_do_verify                           4465	1_1_1	EXIST::FUNCTION:SM2
-SM2_sign                                4466	1_1_1	EXIST::FUNCTION:SM2
-ERR_load_SM2_strings                    4467	1_1_1	EXIST::FUNCTION:SM2
-SM2_plaintext_size                      4468	1_1_1	EXIST::FUNCTION:SM2
-conf_ssl_name_find                      4469	1_1_0i	EXIST::FUNCTION:
-conf_ssl_get_cmd                        4470	1_1_0i	EXIST::FUNCTION:
-conf_ssl_get                            4471	1_1_0i	EXIST::FUNCTION:
+conf_ssl_name_find                      4459	1_1_0i	EXIST::FUNCTION:
+conf_ssl_get_cmd                        4460	1_1_0i	EXIST::FUNCTION:
+conf_ssl_get                            4461	1_1_0i	EXIST::FUNCTION:
```

Comment:
We must remember to update 1.1.0's `util/libcrypto.num` accordingly as well.  They *must* match.

Reply:
I could just remove the SM2 functions and leave a gap in the assignments if that is preferable. TBH I was not sure what your policy is here or if renumbering was appropriate.

Reply:
I personally prefer to close the gaps when possible.  The *technical requirement* is that only unreleased symbols get moved.  Since we haven't released 1.1.0i yet, renumbering these is fine, as long as we do the same renumbering in all affected branches (i.e. in the 1.1.0 branch as well).